### DISCUSSION TITLE
2022/02/17/Hexo-Butterfly-%E5%BB%BA%E7%AB%99%E6%8C%87%E5%8D%97%EF%BC%88%E4%B8%89%EF%BC%89Butterfly-%E4%B8%BB%E9%A2%98/

### DISCUSSION THREAD

**Discussion by giscus** [nickxudotme/nickxudotme.github.io#19] [General]
# 2022/02/17/Hexo-Butterfly-%E5%BB%BA%E7%AB%99%E6%8C%87%E5%8D%97%EF%BC%88%E4%B8%89%EF%BC%89Butterfly-%E4%B8%BB%E9%A2%98/

本篇内容主要源于：  Butterfly 官方安装文档 Hexo+Butterfly主题美化 | 唐先森の博客   安装 在你的博客根目录下载主题并安装依赖： 12git clone -b master https:&#x2F;&#x2F;github.com&#x2F;jerryc127&#x2F;hexo-theme-butterf

https://nickxu.me/2022/02/17/Hexo-Butterfly-%E5%BB%BA%E7%AB%99%E6%8C%87%E5%8D%97%EF%BC%88%E4%B8%89%EF%BC%89Butterfly-%E4%B8%BB%E9%A2%98/

**Comment by Punny520** [comment]
求字体

**Reply by nickxudotme** [reply]
字体劳烦 F12 自行爬一下～
